src/VisualStudio/Core/Def/Implementation/ProjectSystem/AbstractProject.cs

```diff
@@ -328,7 +288,7 @@ public ProjectInfo CreateProjectInfoForCurrentState()
                 analyzerReferences: _analyzers.Values.Select(a => a.GetReference()),
                 additionalDocuments: _additionalDocuments.Values.Select(d => d.GetInitialState()));
 
-            return info.WithHasAllInformation(_intellisenseBuildSucceeded);
```

Comment:
@jasonmalinowski @davkean `_intellisenseBuildSucceeded` is set by [IIntellisenseBuildTarget.SetIntellisenseBuildResult](http://source.roslyn.io/#Microsoft.VisualStudio.LanguageServices/Implementation/ProjectSystem/AbstractProject_IIntellisenseBuildTarget.cs,25), and used to create the ProjectInfo for the project. This interface is specific to native project system, so I have moved its implementation to AbstractLegacyProject and abstracted the bool with `DesignTimeBuildStatus`. We will probably need an API on IProjectContext to set this flag - for now, its always set to true for CPS projects.